pipert/core/mini_logics.py

```diff
@@ -20,8 +20,22 @@ def __init__(self, stream_address, queue, fps=30., *args, **kwargs):
         # self.stream = cv2.VideoCapture(self.stream_address)
         self.queue = queue
         self.fps = fps
+        self.updatedConfig = {}
+
+    def changeSourceStream(self):
```

Comment:
@sionovd nice idea changing the capture parameters and then calling `setup` again. Maybe move the functionality of setup (setting up the` VideoCapture` object) to another method and call that method from both the `update` and `setup` methods, in case we will add things to `setup` that we don't want to do all over again for every source stream update.

Reply:
Yup, sounds good.